gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/BakeController.groovy

```diff
@@ -43,9 +44,17 @@ class BakeController {
     bakeService.bakeOptions(cloudProvider)
   }
 
+  @RequestMapping(value = "/api/v1/{region}/logs/{statusId}", method = RequestMethod.GET)
```

Comment:
I have not-so-great feelings about this url. I mirrored what is defined on rosco.

Would something like `/logs/{region}/{statusId}` look better?